tests/test_2_neurolucida.py

```diff
@@ -297,63 +297,6 @@ def test_unfinished_file():
                         "Hit end of file while consuming a neurite",
                         ":12:error")
 
-
```

Comment:
This portion has been moved in the test file of the mutable API.

Reply:
cool, was going to ask exactly that :)